packages/neon-core/src/wallet/Wallet.ts

```diff
@@ -98,78 +98,60 @@ export class Wallet {
    * Attempts to decrypt Account at index in array.
    * @param index Index of Account in array.
    * @param keyphrase keyphrase
-   * @return Decryption success/failure
+   * @return Promise of the decrypted account.
    */
-  public decrypt(index: number, keyphrase: string): boolean {
+  public decrypt(index: number, keyphrase: string): Promise<Account> {

```

Comment:
Returning Promise<Account> is a breaking change. We should be fixing the method to return Promise<boolean> instead.

Reply:
I don't get what you mean. You mean to return promise or not?

Reply:
Promise<bool> would be the correct fix since that was the original intent.

Reply:
ah i see. I was typing `Promise<bool>` but it was interpreted as markdown syntax

Reply:
Changed.